src/envo/envo-edit.owl

```diff
@@ -6865,29 +6868,25 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000124> <http://purl.obolibrar
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_00000125> (mineral spring)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Mineral_spring"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000125> "A spring whose water contains a significant amount of dissolved minerals, that derive from the rocks through which the water flows."^^xsd:string)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Mineral_spring"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000125> "A spring which  water contains a significant amount of dissolved minerals, that derive from the rocks through which the water flows."^^xsd:string)
```

Comment:
don't complicate with stuff that should be in notes.
'A spring that emits water containing a significant amount of dissolved minerals'.
Of course a purist might want to know what you mean by 'minerals'